server/utils/__tests__/is-protected-entity.spec.js

```diff
@@ -115,11 +115,107 @@ describe('isProtectedEntity', () => {
     expect(result).toBe(false);
   });
 
+  it('should return `true` if entity attribute matches the `lt` rule', () => {
+    const entity = { attribute: 1 };
+    const rules = [['attribute', 'lt', 2]];
+    const result = isProtectedEntity(entity, rules);
+
+    expect(result).toBe(true);
+  });
+
+  it('should return `false` if entity attribute does not match the `lt` rule', () => {
+    const entity = { attribute: 1 };
+
+    expect(isProtectedEntity(entity, [['attribute', 'lt', 1]])).toBe(false);
+    expect(isProtectedEntity(entity, [['attribute', 'lt', 0]])).toBe(false);
+  });
+
+  it('should return `true` if entity attribute matches the `lte` rule', () => {
+    const entity = { attribute: 1 };
+
+    expect(isProtectedEntity(entity, [['attribute', 'lte', 1]])).toBe(true);
+    expect(isProtectedEntity(entity, [['attribute', 'lte', 2]])).toBe(true);
+  });
+
+  it('should return `false` if entity attribute does not match the `lte` rule', () => {
+    const entity = { attribute: 1 };
+    const rules = [['attribute', 'lte', 0]];
+    const result = isProtectedEntity(entity, rules);
+
+    expect(result).toBe(false);
+  });
+
+  it('should return `true` if entity attribute matches the `gt` rule', () => {
+    const entity = { attribute: 1 };
+    const rules = [['attribute', 'gt', 0]];
+    const result = isProtectedEntity(entity, rules);
+
+    expect(result).toBe(true);
+  });
+
+  it('should return `false` if entity attribute does not match the `gt` rule', () => {
+    const entity = { attribute: 1 };
+    expect(isProtectedEntity(entity, [['attribute', 'gt', 1]])).toBe(false);
```

Comment:
@BorysShulyak Can you include a space between the `const` line and the `expect` line?

Reply:
Done.